### PR TITLE
Add support for drag mouse event

### PR DESCRIPTION
**Pull request by ARDiDo**
I wasn't too sure which of our default mousebindings should have been changed to "Drag", so I changed all of the resize and move related mousebindings.

### CODE REVIEW

**Comment by johanmalm** [comment]
Looking at openbox’s rc.xml, should Raise and Focus happen on Press, and Move/Resize happen on Drag?

I think that would feel more natural too.

https://github.com/danakj/openbox/blob/master/data/rc.xml

File: docs/labwc-config.5.scd [hunk 1]
```diff
@@ -144,6 +144,7 @@ Configuration must be wrapped in a <labwc_config> root-node.
 	- Release: Releasing the specified button in the context.
 	- Click: Pressing and then releasing inside of the the context.
 	- DoubleClick: Two presses within the doubleClickTime.
+	- Drag: Pressing the button within the contex, then moving the cursor
```
Comment (johanmalm) [inline docs/labwc-config.5.scd:147]: Missing t in context

**Comment by ARDiDo** [comment]
That should do it. What do you think @johanmalm?

**Comment by johanmalm** [comment]
Looking good. Let's go for it :smile:

**Comment by johanmalm** [comment]
I've pulled the "Unmaximize on move" PR too.
Running with defaults (without config file), there if something odd going on... when double-clicking the title-bar it maximizes, but then maximizes as soon as your move the mouse.
When double-clicking the left/right/top/bottom edges, it's the same (get stuck on resizing).
It's getting a bit late here, so will leave it.
Appreciate anyone investigating.

**Comment by Consolatis** [comment]
Had a quick test of #202 using `build/labwc -c /dev/null` and it seems to fix it.

**Pull request merged**